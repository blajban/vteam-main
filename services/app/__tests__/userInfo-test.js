import React from 'react';
import renderer from 'react-test-renderer';
import UserInfo from '../components/userInfo';

const tree = renderer.create(<UserInfo />).toJSON();



test('Userinfo Renders correctly', () => {
    expect(tree).toMatchSnapshot();
});