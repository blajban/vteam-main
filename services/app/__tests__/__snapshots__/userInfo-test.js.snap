// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Userinfo Renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "borderColor": "#228377",
      "borderRadius": 10,
      "borderWidth": 5,
      "flex": 1,
      "marginBottom": 30,
      "width": "80%",
    }
  }
>
  <View
    style={
      Object {
        "flex": 1,
        "padding": 10,
      }
    }
  >
    <Text
      style={
        Object {
          "fontSize": 40,
          "marginRight": "auto",
          "textDecorationLine": "underline",
        }
      }
    >
      FÃ¶rnamn
    </Text>
    <Text>
      Efternamn
    </Text>
  </View>
  <View
    style={
      Object {
        "backgroundColor": "#84E1D6",
        "borderRadius": 10,
        "flex": 1,
        "height": 80,
        "margin": 10,
        "padding": 10,
        "width": "90%",
      }
    }
    testID="saldoContainer"
  >
    <Text>
      Ditt saldo
    </Text>
    <Text
      style={
        Object {
          "fontSize": 40,
          "fontWeight": "bold",
          "marginRight": "auto",
        }
      }
    >
      235 kr
    </Text>
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "position": "absolute",
          "right": 0,
        }
      }
    >
      <Image
        source={1}
        style={
          Object {
            "marginRight": 5,
            "marginTop": 5,
            "position": "absolute",
            "right": 0,
          }
        }
      />
    </View>
  </View>
</View>
`;
